README.md

```diff
@@ -88,7 +88,109 @@ print(tokenizer.decode(outputs[0], skip_special_tokens=True))
 **Note:** You may need to tune the Temperature setting  for different applications. Typically, a lower Temperature is helpful for tasks that require deterministic outcomes. 
 Additionally, for tasks demanding adherence to specific formats or function calls, explicitly including formatting instructions is advisable and important. 
 
+# Deploying and Interacting with xLAM Models
 
+There are two main options for serving the xLAM model as an OpenAI-compatible chat completion API (here we use `Salesforce/xLAM-8x7b-r` as an example):
+
+## Option 1: Using vLLM (Recommended)
+
+vLLM offers efficient serving with lower latency. To serve the model with vLLM:
+
+```bash
+python -u -m vllm.entrypoints.openai.api_server \
```

Comment:
FYI in the latest version of `vllm`, the [API](https://docs.vllm.ai/en/latest/serving/openai_compatible_server.html) has changed somewhat and now one should use:

```
vllm serve model Salesforce/xLAM-8x7b-r --host 0.0.0.0 --tensor-parallel-size 2 ...
```

Reply:
Fixed. Thanks @lewtun!